tests/import.test.ts

```diff
@@ -11,24 +12,26 @@ interface TestDefinition {
   message: string
   inputFile: string
   guardFile: string
+  only?: true
 }
 
 // Test blueprint for running different test definitions
 class Blueprint {
-  inputContents: string
-  expectedContents: string
-  message: string
-  constructor(message: string, inputFile: string, guardFile: string) {
-    this.inputContents = inputFile
-    this.expectedContents = guardFile
-    this.message = message
-  }
+  constructor(
+    public message: string,
```

Comment:
This is cool! I haven't used TypeScript for ages so I never got to use this one. :-)